backend/test/test_uploads.py

```diff
@@ -252,6 +252,8 @@ def test_get_upload_pages(admin_auth_headers, default_org_id, upload_id):
         assert page["crawl_id"] == upload_id
         assert page["url"]
         assert page["ts"]
+        assert page["filename"]
+        assert page["depth"] is not None
```

Comment:
```suggestion
        assert page.get("depth") or page.get("depth") is None
```

^ The WACZs we use as test fixtures for upload are quite old and don't have a lot of the information in the page list that we'd expect them to have. Might be worth updating (at least one of) those to newer examples, but in the meantime we can't expect to get depth from them

Reply:
ah right, uploads would not have this anyway actually.. maybe should default depth to 0, and remove this check

Reply:
can remove these checks, not sure about setting default depth to 0 - I think it's better to be lacking information than give the false impression everything is a seed?